assemblies/customizing-component-deployment-resources.adoc

```diff
@@ -0,0 +1,22 @@
+:_module-type: ASSEMBLY
+
+ifdef::context[:parent-context: {context}]
+
+:context: custom-resource
+
+[id="customizing-component-deployment-resources"]
```

Comment:
ID missing the _{context} at the end

[id="customizing-component-deployment-resources_{context}"]

Reply:
Good spot, updated!